### ISSUE TITLE
`dir` should allow multiple

### ISSUE THREAD

**Issue by joscha** [lipp/doclets#77]
I get the error:

```
Error: TypeError: Path must be a string. Received [ 'src', 'finders' ]
```
but my project has source files that are both in `src` and `finders`. I propose `dir` should be able to accept multipel values.

**Comment by lipp** [comment]
Agree! Thanks for the input! `dir` will accept a string or an array of strings.

**Comment by zandaqo** [comment]
I believe using glob patters is a better solution here. In the long it's only a matter of time when someone decides that having multiple dirs isn't enough and he needs to include/exclude some files according to a patter. Allowing to use glob patterns in `files` parameter would solve this. I believe this PR might work #81
